cmd/zstreamdump/zstreamdump.c

```diff
@@ -265,6 +266,7 @@ main(int argc, char *argv[])
 		exit(1);
 	}
 
+	(void) libzfs_init();
```

Comment:
Can we just use `fletcher_4_init()` / `fletcher_4_fini()` as a more targeted fix?  This file already directly includes `zfs_fletcher.h`.  It would nice to not have all the additional `libzfs_init()` junk.